---
author-meta:
- J. Brian Byrd
- Weixuan Fu
- Casey Greene
- Daniel Himmelstein
- Qiwen Hu
- Lawrence Hunter
- Shantanu Jain
- Adam Kurkiewicz
- Trang Le
- Binglan Li
- Jason Miller
- Luca Pinello
- Rashika Ramola
- Jaclyn Taroni
- Yihsuan Tsai
- Robin van der Lee
- Ryan Whaley
date-meta: '2019-01-07'
keywords:
- manubot
- tutorial
- conference
- psb19
- pacific symposium on biocomputing
lang: en-US
title: "The people of the 24\u1D57\u02B0 annual Pacific Symposium on Biocomputing\
  \ in Hawaii"
...






<small><em>
This manuscript
([permalink](https://dhimmel.github.io/psb-manuscript/v/6a49ed54d66524dd0f9eaba90308ebdc38983e6b/))
was automatically generated
from [dhimmel/psb-manuscript@6a49ed5](https://github.com/dhimmel/psb-manuscript/tree/6a49ed54d66524dd0f9eaba90308ebdc38983e6b)
on January 7, 2019.
</em></small>

## Authors



+ **J. Brian Byrd**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0002-0509-3520](https://orcid.org/0000-0002-0509-3520)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [byrdjb](https://github.com/byrdjb)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [thebyrdlab](https://twitter.com/thebyrdlab)<br>
  <small>
     University of Michigan Medical School
  </small>

+ **Weixuan Fu**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0002-6434-5468](https://orcid.org/0000-0002-6434-5468)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [weixuanfu](https://github.com/weixuanfu)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [weixuanfu](https://twitter.com/weixuanfu)<br>
  <small>
     Department of Biostatistics, Epidemiology and Informatics, Institute for Biomedical Informatics, University of Pennsylvania
  </small>

+ **Casey Greene**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0001-8713-9213](https://orcid.org/0000-0001-8713-9213)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [cgreene](https://github.com/cgreene)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [greenescientist](https://twitter.com/greenescientist)<br>
  <small>
     Department of Systems Pharmacology & Translational Therapeutics, University of Pennsylvania; Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation
  </small>

+ **Daniel Himmelstein**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0002-3012-7446](https://orcid.org/0000-0002-3012-7446)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [dhimmel](https://github.com/dhimmel)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [dhimmel](https://twitter.com/dhimmel)<br>
  <small>
     Department of Systems Pharmacology & Translational Therapeutics, University of Pennsylvania
  </small>

+ **Qiwen Hu**<br>
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [huqiwen0313](https://github.com/huqiwen0313)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [qiwen_hu](https://twitter.com/qiwen_hu)<br>
  <small>
     Department of Systems Pharmacology & Translational Therapeutics, University of Pennsylvania
  </small>

+ **Lawrence Hunter**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0003-1455-3370](https://orcid.org/0000-0003-1455-3370)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [LEHunter](https://github.com/LEHunter)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [ProfLHunter](https://twitter.com/ProfLHunter)<br>
  <small>
     University of Colorado Anschutz School of Medicine; University of Colorado Boulder
  </small>

+ **Shantanu Jain**<br>
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [shajain](https://github.com/shajain)<br>
  <small>
     Northeastern University
  </small>

+ **Adam Kurkiewicz**<br>
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [picrin](https://github.com/picrin)<br>
  <small>
     University of Glasgow
  </small>

+ **Trang Le**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0003-3737-6565](https://orcid.org/0000-0003-3737-6565)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [trang1618](https://github.com/trang1618)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [trang1618](https://twitter.com/trang1618)<br>
  <small>
     University of Pennsylvania
  </small>

+ **Binglan Li**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [None](https://orcid.org/None)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [BinglanLi](https://github.com/BinglanLi)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [victoria_bll](https://twitter.com/victoria_bll)<br>
  <small>
     University of Pennsylvania
  </small>

+ **Jason Miller**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0002-1835-3512](https://orcid.org/0000-0002-1835-3512)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [git-jemiller](https://github.com/git-jemiller)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [JEMgenes](https://twitter.com/JEMgenes)<br>
  <small>
     Department of Genetics, Institute for Biomedical Informatics, and Perelman School of Medicine, University of Pennsylvania
  </small>

+ **Luca Pinello**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0003-1109-3823](https://orcid.org/0000-0003-1109-3823)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [pinellolab](https://github.com/pinellolab)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [lucapinello](https://twitter.com/lucapinello)<br>
  <small>
     Massachusetts General Hospital/ Harvard Medical School
  </small>

+ **Rashika Ramola**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [None](https://orcid.org/None)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [Rashika40](https://github.com/Rashika40)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [None](https://twitter.com/None)<br>
  <small>
     Northeastern University
  </small>

+ **Jaclyn Taroni**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0003-4734-4508](https://orcid.org/0000-0003-4734-4508)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [jaclyn-taroni](https://github.com/jaclyn-taroni)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [jaclyn_taroni](https://twitter.com/jaclyn_taroni)<br>
  <small>
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation
  </small>

+ **Yihsuan Tsai**<br>
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [ShannonSzeto](https://github.com/ShannonSzeto)<br>
  <small>
     University of North Carolina at Chapel Hill
  </small>

+ **Robin van der Lee**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0001-7391-9438](https://orcid.org/0000-0001-7391-9438)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [robinvanderlee](https://github.com/robinvanderlee)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [robinvdlee](https://twitter.com/robinvdlee)<br>
  <small>
     The University of British Columbia, Vancouver, Canada; BC Children’s Hospital Research Institute; Centre for Molecular Medicine and Therapeutics
  </small>

+ **Ryan Whaley**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0002-8810-3893](https://orcid.org/0000-0002-8810-3893)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [whaleyr](https://github.com/whaleyr)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [ryanatpharmgkb](https://twitter.com/ryanatpharmgkb)<br>
  <small>
     Stanford University
  </small>



## Abstract {.page_break_before}

Manubot is an open source tool for writing manuscripts on GitHub in markdown format. Manubot applies the git-based software workflow to scholarly writing, enabling enhanced transparency, collaboration, automation, and reproducibility. 

This manuscript is the result of a _special working group_ at the 2019 [Pacific Symposium on Biocomputing](https://psb.stanford.edu/) that will introduce attendees to collaborative writing with Manubot.
Each conference attendee is invited to write a small blurb on themselves and their research, by submitting a pull request to the manuscript repository at <https://github.com/dhimmel/psb-manuscript>.

The working group also [covers](https://slides.com/dhimmel/psb) how to write your next manuscript [using Manubot](https://manubot.org) and what features of Manubot can help biomedical researchers document and publish their computational research.
For example, Manubot enables citation by persistent identifier to automate bibliographic metadata retrieval and formatting as well as allowing templating so results can be directly inserted from the analyses that produced them.


## Methods {.page_break_before}

In this section, PSB 2019 attendees are asked to contribute a bit about themselves and their research.
As part of the [special working group](http://web.archive.org/web/20190103203407/https://psb.stanford.edu/working%20group/), we thought this would be a helpful activity to introduce biocomputational scientists to writing with Manubot.
For inspiration, here are some prompts:

- Introduce yourself briefly.
- What do you research? Include any relevant links to your lab or personal website.
- What is your favorite study from your career or what study was your biggest discovery?
- What was your first scholarly publication?
- Add a code snippet or mathematical equation.
- Add a figure with a caption. This could be a picture of you in Hawaii or a figure from your previous work if the license is permissive enough to allow reuse.

Self-citations are explicitly encouraged, since one goal of this activity is to introduce attendees to the concept of [citation by persistent identifier](https://github.com/dhimmel/psb-manuscript/blob/master/USAGE.md#citations).
By having attendees cite their existing works, we hope to show researchers how references can be created from just persistent identifiers, and how this can assist with collaborative and transparent authoring.

The [markdown manuscript source](https://github.com/dhimmel/psb-manuscript/tree/master/content) has a section for each PSB 2019 attendee, generated from the online [attendee list](https://github.com/dhimmel/psb-manuscript/blob/master/attendees/attendees.pdf).
Names are ordered alphabetically by last name.
If you'd like to contribute, but are not already in the list, please insert your section at the appropriate alphabetical location.

For questions on how to use Manubot, see the [usage documentation](https://github.com/dhimmel/psb-manuscript/blob/master/USAGE.md).
More information on the tool and its inception is available in the project manuscript [@1GGGHdsew].


## Attendees {.page_break_before}

<!-- Max Alekseyev -->



<!-- Russ Altman -->



<!-- Kiyoshi Asai -->



<!-- Folkert Asselbergs -->



<!-- Ho Bae -->



<!-- Brett Beaulieu-Jones -->



<!-- George Bebis -->



<!-- Joanne Berghout -->



<!-- John Black -->



<!-- Carly Bobak -->



<!-- Mary Boland -->



<!-- Philip Bourne -->



<!-- Soline Boussard -->



<!-- Steven Brenner -->



<!-- Soren Brunak -->



<!-- Martha Bulyk -->



<!-- William Bush -->



<!-- J. Brian Byrd -->

### J. Brian Byrd

I'm a physician-scientist at the University of Michigan.
My laboratory focuses on identifying novel biomarkers for a clinically important subtype of high blood pressure, called primary aldosteronism.
Our principal interest is in detecting the transcriptional activity of the mineralocorticoid receptor [@GGRHirwN].

<!-- Bin Chen -->



<!-- Rachel Chen -->



<!-- Yong Chen -->



<!-- Jane Chiang -->



<!-- Jongmun Choi -->



<!-- DongWon Choo -->



<!-- Christopher Chute -->



<!-- Leonardo Collado Torres -->



<!-- Christian Darabos -->



<!-- Supriyo De -->



<!-- Jessica De Freitas -->



<!-- Scott Delp -->



<!-- Emek Demir -->



<!-- Daisy Yi Ding -->



<!-- Valentin Dinu -->



<!-- Megan Doerr -->



<!-- Ernst Dow -->



<!-- Sorin Draghici -->



<!-- A Keith Dunker -->



<!-- Claire Duvallet -->



<!-- Prashant Emani -->



<!-- Peter Embi -->



<!-- Barbara Engelhardt -->



<!-- Yayin Fang -->



<!-- Kimberley Ferguson -->



<!-- James Foster -->



<!-- Verena Friedl -->



<!-- Weixuan Fu -->

### Weixuan Fu

Aloha, I'm in the [Institute for Biomedical Informatics (IBI)](http://upibi.org/) at the University of Pennsylvania and the developer of [TPOT](https://github.com/EpistasisLab/tpot) and PennAI [@MARd00zz].

My main interest of research is developing automated machine learning tools for the analysis of large scale biomedical/sequencing data.
Besides that, I am working on optimizing analysis pipeline of predicting neoantigen specifically presented in tumor cells using DNA and RNA sequencing data, for designing personalized neoantigen vaccines in cancer immunotherapies.


<!-- Tetsu Furukawa -->



<!-- Lana Garmire -->



<!-- Maxwell Gold -->



<!-- Graciela Gonzalez Hernandez -->



<!-- Raluca Gordan -->



<!-- Max Gordon -->



<!-- Kiley Graim -->



<!-- Casey Greene -->

### Casey Greene

I run an integrative genomics research lab at the University of of Pennsylvania, and I direct the Childhood Cancer Data Lab for Alex's Lemonade Stand Foundation.
The lab at Penn develops methods to integrate large-scale public datasets, primarily from transcriptomic assays, and applies these methods to a broad set of biological questions.
We've studied numerous systems, and we currently have active research projects in the application areas of microbial systems [@1CFhfCyWN; @Hlprh8TG], cancers [@7V2oCvtf; @17XHtmqPY; @X6oQhIf8; @11ZPgRuLk], and rare diseases [@1GtRgPRxn].
At this PSB, a postdoc from the group will present a paper describing Continental Breakfast Included (CBI) effect in the final talk of the final session of this year's meeting [@5CsWRjfp].

I'm also interested in technologies that enhance the future of scientific communication.
Our lab has studied Sci-Hub [@IhliSZDo].
We've led a large collaborative review of deep learning in biology and medicine [@PZMP42Ak].
Members of the lab have developed tools like manubot [@1GGGHdsew], which you are using now.
More publications are available on our [lab website](http://www.greenelab.com/publications).

<!-- Peyton Greenside -->



<!-- Gamze Gursoy -->



<!-- Jonathan Haines -->



<!-- Greg Hampikian -->



<!-- Jiali Han -->



<!-- Wontack Han -->



<!-- David Haussler -->



<!-- Tina Hernandez-Boussard -->



<!-- Daniel Himmelstein -->

### Daniel Himmelstein

Greetings, I'm in the [Greene Lab](http://www.greenelab.com/) at the University of Pennsylvania and am the lead developer of the Manubot project.
2019 is my first PSB and I'm exciting to backpack around the Big Island following the conference.

My main area of research is integrating biomedical knowledge using hetnets [@XJuQlZl1; @O21tn8vf].
However, I've also studied Sci-Hub, finding that it provides access to nearly all paywalled scholarly literature [@vkig9eAb].
Perhaps my biggest discovery was observing an epidemiological association that higher elevation counties have lower rates of lung cancer, suggesting that oxygen is an inhaled carcinogen (Figure @fig:elevcan) [@17wiYglmF; @8YjPCnYe].

![
The association between elevation and lung cancer across Western U.S. counties.
This figure is reused from [here](https://doi.org/10.7717/peerj.705/fig-4) under its CC BY 4.0 License.
](https://github.com/dhimmel/elevcan/raw/7aed9f29d2371eb4918f337a138608e6b6d9e311/manual/figures/peerj/Figure_4.png){#fig:elevcan width="100%"}

I haven't done much text mining, but I did enjoy extracting attendee names for PSB from the online PDF.
Converting the PDF to text in Python was [as easy as](https://github.com/dhimmel/psb-manuscript/blob/15babecdf2a915f88088703e23a61e34e1294b1f/attendees/attendees.ipynb):

```python
# https://stackoverflow.com/a/48673754
import tika.parser
parsed = tika.parser.from_file('attendees.pdf')
text = parsed["content"]
```

<!-- Michael Hoffman -->



<!-- John Holmes -->



<!-- Rachel Hovde -->



<!-- Qiwen Hu -->
### Qiwen Hu

I'm a postdoc from [Greene Lab](http://www.greenelab.com/) at the University of Pennsylvania.
My research focus on integrating different types of high-throughput sequencing data to find meaningful biological signals behind it.
I developed machine learning and statistical approaches to identify regulatory elements that affect transcription and translation.
I also developed machine learning-based methods to extract regulatory signals from addicted brain [@1m3Egymh], developmental tissues [@M0crDyWe] and cell-type signals from single-cell datasets.

This year at PSB, I will present our findings for analyzing single-cell data based on deep variation auto-encoders [@5CsWRjfp].

<!-- Zhiyue Hu -->



<!-- Lawrence Hunter -->

### Lawrence Hunter 

I'm a cofounder of the [PSB conference](http://psb.stanford.edu), and a professor at the University of Colorado School of Medicine.
You can find information about my lab at <http://compbio.ucdenver.edu/Hunter>.
One of my early papers is @mFhozBp8.

<!-- Shantanu Jain -->

### Shantanu Jain

Hi all, I am very excited to be here attending PSB.
I am a research scientist at Northeastern University.
I am broadly interested in Machine learning methods.
During my Ph.D., I worked on positive unlabeled learning.
I am most proud about my research on non parametric estimation of class priors from positive and unlabeled data [@BYxifaL7].
I have started learning about Causal Inference lately and I am interested in applying it to Biological datasets.

<!-- Yuexu Jiang -->



<!-- Kipp Johnson -->



<!-- Kory Johnson -->



<!-- Indika Kahanda -->



<!-- Laurynas Kalesinskas -->



<!-- Michael Keiser -->



<!-- Andreas Keller -->



<!-- Colleen Kenost -->



<!-- Aly Khan -->



<!-- Dokyoon Kim -->



<!-- Jeremie Kim -->



<!-- Teri Klein -->



<!-- Kord Kober -->



<!-- Milica Krunic -->



<!-- Adam Kurkiewicz -->

### Adam Kurkiewicz

I'm interested in building a tool to do SNP calling from single cell RNASeq data.
This has been tried before by various groups, e.g. check out the honeyBADGER paper [@1elWGdf0], but ultimately none of the approaches were successful.
I have a few ideas on how to make progress --- give me a shout if you'd like to discuss!


<!-- Nicholas Larson -->



<!-- Janet Layne -->



<!-- Trang Le -->

### Trang Le

Hello from the [Moore lab](http://epistasis.org/) at the University of Pennsylvania!
I'm [a mathematician](http://lelaboratoire.github.io) who's currently excited about automated machine learning.

Here goes the self-citations:

- My own favorite study: Generalization of the Fermi Pseudopotential [@10GH4uYUR] - a piece of mathematical physics work I got to do when procrastinating writing my dissertation.
- My first (first-author) scholarly publication: Differential privacy-based evaporative cooling feature selection and classification with relief-F and random forests [@hUoMaAJ6].
Check out the Github repo for this study [here](https://github.com/insilico/privateEC).
- Code snippet I'm most proud of:
  ```
  M = dec2bin(0:2^(n*n)-1,n*n)
  ```
  I will be impressed if you could tell what the language is.
  This is my answer to a question on [Math StackExchange](https://math.stackexchange.com/questions/1943862/matlab-code-for-an-array-consisting-of-matrices/1943923#1943923).
- I have too many favorite mathematical equations, but here's one:
  $$a^p \equiv a \mod p$$
  Anyone recognize this theorem?
- And a figure with a caption:

  ![
    Participants tended to have older brain-predicted age when given placebo.
  ](https://s3.amazonaws.com/media-p.slid.es/uploads/909204/images/5547949/fig2.png){#fig:ibuBrainAGE width="80%"}

This is an improved version of my main figure in this interesting study [@ih7MBGjv].

<!-- Jinhee Lee -->



<!-- Nicholas Lee -->



<!-- DoHyeon Lee -->



<!-- Binglan Li -->

### Binglan Li

Greeting from the [Ritchie Lab](http://ritchielab.org/) at the University of Pennsylvania.
I am a third year graduate student in the Genomics and Computational Biology programe and interested in prioritization of drug response-related gene via data integration approaches. 

I am still on the early part of research journey.
But I would love to share my latest work published in the PSB 2019 proceedings 2019.

- Influence of tissue context on gene prioritization for predicted transcriptome-wide association studies [@9B8DWkq3].

- Code snippet I'm most proud of:
```
############################################
## Menu
## 1. Food Preparation
## 1.1. Load Necessary Libraries and Scripts
## 1.2. Define Parameters
## 2. Appetizers
## 2.1. Data Simulation
## 2.2. eQTL Detection
## 3. Entree/Main Course
## 3.1. Run single-tissue TWAS
## 3.2. Run integrative TWAS
## 3.3. Evaluate Power and Type I Error Rate of TWAS Results
## 4. Dessert
## 4.1. None. Sorry this is a healthy (aka anti-sweet) restaurant.
############################################

## actual code
set.seed(random_seed, kind = "L'Ecuyer-CMRG")
```

- Here is a plot about minor allele frequency of the eQTLs in the GTEx v7 whole blood tissue.
Please pretend that you do see a title in the figure.

![
Minor allele frequency of eQTLs in the GTEx v7 whole blood tissue.
](https://user-images.githubusercontent.com/1117703/50731107-cd16b380-1100-11e9-930c-3a23cb4a0ec9.png){#fig:binglan width="50%"}

<!-- Haiquan Li -->



<!-- Hua Li -->



<!-- Jianrong Li -->



<!-- Kevin Li -->



<!-- Sooyeon Lim -->



<!-- Hongfang Liu -->



<!-- Zheng Liu -->



<!-- Xueqing Lu -->



<!-- Zhiyong Lu -->



<!-- Yves Lussier -->



<!-- Arjun Magge -->



<!-- Lara Mangravite -->



<!-- Serghei Mangul -->



<!-- Gabor Marth -->



<!-- Rachel Marty -->



<!-- Wouter Meuleman -->



<!-- Jason Miller -->

### Jason E. Miller

Hi, I'm a postdoctoral fellow from the [Ritchie lab](https://ritchielab.org) at the University of Pennsylvania. 

I'm currently focussed on identifying how genetic variation leads to Alzheimer's disease through perturbation of gene regulatory mechanisms.

My favorite study from my career identified specific types of codon bias among synonymous variants, such as those related to codon optimality and frequency, that are associated with an Alzheimer's disease imaging endophenotype [@czfw16Ig].

If you are interested, you can check out my github page [here](https://github.com/git-jemiller).

<!-- Stephen Montgomery -->



<!-- Abigail Moore -->



<!-- Jason Moore -->



<!-- Alexander Morgan -->



<!-- Sorana Morrissy -->



<!-- Meghan Muse -->



<!-- Rikke Nielsen -->



<!-- William Noble -->



<!-- Soichi Ogishima -->



<!-- Lawrence Oloff -->



<!-- Zhengqing Ouyang -->



<!-- Philip Payne -->



<!-- Matteo Pellegrini -->



<!-- Luca Pinello -->

### Luca Pinello

Aloah from the [Pinello Lab!](http://pinellolab.org/)

I am a computational biologist studying the role of chromatin structure/dynamics and non-coding regions including enhancers, promoters, insulators and their role in gene regulation. 
The mission of my lab is the integration of omics data to explore and better understand the functional mechanisms of the non-coding genome and to provide accessible tools for the community to accelerate discovery in this field. 
The long-term goal of my research is to develop innovative computational approaches and to use cutting-edge experimental assays, such as single cell and genome editing, to systematically analyze sources of genetic and epigenetic variation that affect gene regulation in different human traits and diseases. 
I believe this will further our understanding of disease etiology involving these poorly characterized regions and will provide a foundation for the development of new drugs and more targeted treatments.

I am excited to share during the workshop [Reading between the genes: Interpreting noncoding DNA in high throughput](http://lussierlab.net/pacific-symposium-2019/index.html) a new computational methods we have recently developed to analyze CRISPR tiling screen called CRISPR-SURF. 
You can read more on the manuscript that was recently published in Nature Methods [@WXZIineW]. 

<!-- Niranjani Prasad -->



<!-- Blake Pyman -->



<!-- Tielin Qin -->



<!-- Aaron Quinlan -->



<!-- Predrag Radivojac -->



<!-- Rashika Ramola -->

### Rashika Ramola

Hi I am Rashika Ramola.
I am a PhD student at Northeastern University.
This is my first PSB.
I like computational biology, and I am excited to be here.

My first paper studies some performance measures (accuracy, balanced accuracy, f-measure and Matthews Correlation Coefficient) in positive-unlabeled learning [@10OhwB9If].
In this work, we demonstrate how performance measure can be inaccurate in positive unlabeled setting, and then we introduce correction measures.

I am including an important formula from the abovementioned mauscript:

$$
\mcc=\frac{1}{\mpl-\mpu}\sqrt{\frac{\mpt\paren*{1-\mpt}}{\propL(1-\propL)}}\cdot \mccPU
$$

It shows that Matthews correlation coefficient (MCC) is directly proportional to its equivalent in positive unlabeled setting.
Thus, MCC is a well behaved performance measure.

Here is a beautiful aerial shot of [Hawaii](https://www.myhawaii.com.au/wp-content/uploads/sites/13/2018/08/Hawaii-Landscape-Copy.jpg).

<!-- Jayamary Divya Ravichandar -->



<!-- Derek Reiman -->



<!-- Haluk Resat -->



<!-- Marylyn Ritchie -->



<!-- David Rocke -->



<!-- Pratyaydipta Rudra -->



<!-- Indra Sarkar -->



<!-- Sebastian Schaaf -->



<!-- Ralph Schlapbach -->



<!-- Christine Scholberg -->



<!-- Ashwini Sehgal -->



<!-- Martin Seneviratne -->



<!-- Matthew Settles -->



<!-- James Sikela -->



<!-- Sean Simmons -->



<!-- Marina Sirota -->



<!-- Giltae Song -->



<!-- Paul Spellman -->



<!-- Arunima Srivastava -->



<!-- Josh Stuart -->



<!-- Yoichi Takenaka -->



<!-- Eric Talevich -->



<!-- Haixu Tang -->



<!-- Lin Tang -->



<!-- Yosuke Tanigawa -->



<!-- Cui Tao -->



<!-- Jaclyn Taroni -->

### Jaclyn Taroni

I'm a data scientist at the [Childhood Cancer Data Lab](https://www.ccdatalab.org) (CCDL), an initiative of [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org).
I'm interested in how diverse collections of publicly available transcriptomic data can help us learn about the biology of rare diseases. 
As a graduate student, I studied systemic sclerosis [@1Av3x1pnk].
In the [PSB 2019 Text Mining and Machine Learning for Precision Medicine Workshop](https://healthlanguageprocessing.org/psb-2019-tm-ml-pm-workshop/), I'll present our MultiPLIER project [@1GtRgPRxn].
With the CCDL, I've been working on [refine.bio](https://www.refine.bio), a project for uniformly processing transcriptomic data from multiple species.

<!-- Gunjan Thakur -->



<!-- Robert Thurman -->



<!-- Nathan Tintle -->



<!-- Yihsuan Tsai -->

### Yihsuan Tsai (Shannon)

This is Shannon from UNC at Chapel Hill.
I'm a bioinformatics scientis at [UNC lineberger cancer center](https://lbc.unc.edu/). 
My recent research project could be found at PSB poster section #69.
It's about using methylation data to predict tumor infiltrating lymphocytes, which is highly correlated with patient survival in Melanoma. 

Here are some of my publications:

1. Meta-analysis of airway epithelium gene expression in asthma [@r7HvVKdn].
2. Identification of a robust methylation classifier for cutaneous melanoma diagnosis [@yKr4neGw].
3. Transcriptome-wide identification and study of cancer-specific splicing events across multiple tumors [@9sfG2U7T].
4. Prevalent RNA recognition motif duplication in the human genome [@jk5tACP8].

<!-- Brian Yik Tak Tsui -->



<!-- Anders Ulrik Eliasen -->



<!-- Paul Utz -->



<!-- Robin van der Lee -->

### Robin van der Lee

Hi! I'm a post-doc with Wyeth Wasserman at UBC, Vancouver, Canada.
Info about the lab can be found at <http://www.cisreg.ca> and <https://github.com/wassermanlab>.

My PhD work was on integrative omics to discover genes involved in immunity [@6zDUsoSA].
I also did some work on comparative genomics of primate genomes, finding that rapidly evolving genes are predictive of virus-human interactions [@Myj0Ouv5].

In my post-doc work, I am developing methods for interpreting regulatory genomic variants based on alterations to transcription factor binding motifs.
Some of that work is on **poster 71**, which I will present on Saturday 5 January 2018 at the PSB meeting.

![
This is the header of the poster I'll present here under its CC BY 4.0 License.
](images/Robin-van-der-Lee_Poster_Pacific_Symposium_on_Biocomputing_PSB_2019__MANTA-RAE_poster_header.jpg){#fig:vanderlee width="100%"}

<!-- Maya Varma -->



<!-- Anurag Verma -->



<!-- Shefali Verma -->



<!-- Yogasudha Veturi -->



<!-- Francesca Vitali -->



<!-- Dennis Wall -->



<!-- Duolin Wang -->



<!-- Haohan Wang -->



<!-- Jing Wang -->



<!-- Tongxin Wang -->



<!-- Yong Wang -->



<!-- Joanne Watson -->



<!-- Ryan Whaley -->

### Ryan Whaley

Hi, I'm Ryan and I'm one of the technical leads for [PharmGKB](https://www.pharmgkb.org).
I'm also helping to run the A/V desk during this presentation.

I'm trained in software development and started by career as a DBA.
Over the past decade I've switched to Java and then web application development.
I've contributed to PharmGKB [@3NKcTIrb], CPIC [@1EYG4hAqR], and other PGx consortia.

<!-- Nicholas Wheeler -->



<!-- Michelle Whirl-Carrillo -->



<!-- Scott Williams -->



<!-- John Witte -->



<!-- Yuanxin Xi -->



<!-- Madelyne Xiao -->



<!-- Yuzhen Ye -->



<!-- Shuxing Zhang -->



<!-- Xinyuan Zhang -->



<!-- Yuping Zhang -->



<!-- Zemin Zhang -->



<!-- the end -->


## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
