<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="J. Brian Byrd" />
  <meta name="author" content="Weixuan Fu" />
  <meta name="author" content="Casey Greene" />
  <meta name="author" content="Daniel Himmelstein" />
  <meta name="author" content="Qiwen Hu" />
  <meta name="author" content="Lawrence Hunter" />
  <meta name="author" content="Shantanu Jain" />
  <meta name="author" content="Adam Kurkiewicz" />
  <meta name="author" content="Trang Le" />
  <meta name="author" content="Binglan Li" />
  <meta name="author" content="Jason Miller" />
  <meta name="author" content="Luca Pinello" />
  <meta name="author" content="Rashika Ramola" />
  <meta name="author" content="Jaclyn Taroni" />
  <meta name="author" content="Yihsuan Tsai" />
  <meta name="author" content="Robin van der Lee" />
  <meta name="author" content="Ryan Whaley" />
  <meta name="dcterms.date" content="2019-01-07" />
  <meta name="keywords" content="manubot, tutorial, conference, psb19, pacific symposium on biocomputing" />
  <title>The people of the 24ᵗʰ annual Pacific Symposium on Biocomputing in Hawaii</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(title);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
  </style>
  <link rel="stylesheet" href="github-pandoc.css" />
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_CHTML-full" type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!-- Insert Analytics Script Below -->
  <script>
  </script>
  <!-- End Analytics Script -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">The people of the 24ᵗʰ annual Pacific Symposium on Biocomputing in Hawaii</h1>
</header>
<p><small><em> This manuscript (<a href="https://dhimmel.github.io/psb-manuscript/v/581cbdd48f044ed180b0b9aa4b40f2da102960bb/">permalink</a>) was automatically generated from <a href="https://github.com/dhimmel/psb-manuscript/tree/581cbdd48f044ed180b0b9aa4b40f2da102960bb">dhimmel/psb-manuscript@581cbdd</a> on January 7, 2019. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>J. Brian Byrd</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-0509-3520">0000-0002-0509-3520</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/byrdjb">byrdjb</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/thebyrdlab">thebyrdlab</a><br> <small> University of Michigan Medical School </small></p></li>
<li><p><strong>Weixuan Fu</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-6434-5468">0000-0002-6434-5468</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/weixuanfu">weixuanfu</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/weixuanfu">weixuanfu</a><br> <small> Department of Biostatistics, Epidemiology and Informatics, Institute for Biomedical Informatics, University of Pennsylvania </small></p></li>
<li><p><strong>Casey Greene</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/cgreene">cgreene</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/greenescientist">greenescientist</a><br> <small> Department of Systems Pharmacology &amp; Translational Therapeutics, University of Pennsylvania; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation </small></p></li>
<li><p><strong>Daniel Himmelstein</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-3012-7446">0000-0002-3012-7446</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/dhimmel">dhimmel</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/dhimmel">dhimmel</a><br> <small> Department of Systems Pharmacology &amp; Translational Therapeutics, University of Pennsylvania </small></p></li>
<li><p><strong>Qiwen Hu</strong><br> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/huqiwen0313">huqiwen0313</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/qiwen_hu">qiwen_hu</a><br> <small> Department of Systems Pharmacology &amp; Translational Therapeutics, University of Pennsylvania </small></p></li>
<li><p><strong>Lawrence Hunter</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0003-1455-3370">0000-0003-1455-3370</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/LEHunter">LEHunter</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/ProfLHunter">ProfLHunter</a><br> <small> University of Colorado Anschutz School of Medicine; University of Colorado Boulder </small></p></li>
<li><p><strong>Shantanu Jain</strong><br> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/shajain">shajain</a><br> <small> Northeastern University </small></p></li>
<li><p><strong>Adam Kurkiewicz</strong><br> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/picrin">picrin</a><br> <small> University of Glasgow </small></p></li>
<li><p><strong>Trang Le</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0003-3737-6565">0000-0003-3737-6565</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/trang1618">trang1618</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/trang1618">trang1618</a><br> <small> University of Pennsylvania </small></p></li>
<li><p><strong>Binglan Li</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/None">None</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/BinglanLi">BinglanLi</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/victoria_bll">victoria_bll</a><br> <small> University of Pennsylvania </small></p></li>
<li><p><strong>Jason Miller</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-1835-3512">0000-0002-1835-3512</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/git-jemiller">git-jemiller</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/JEMgenes">JEMgenes</a><br> <small> Department of Genetics, Institute for Biomedical Informatics, and Perelman School of Medicine, University of Pennsylvania </small></p></li>
<li><p><strong>Luca Pinello</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0003-1109-3823">0000-0003-1109-3823</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/pinellolab">pinellolab</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/lucapinello">lucapinello</a><br> <small> Massachusetts General Hospital/ Harvard Medical School </small></p></li>
<li><p><strong>Rashika Ramola</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/None">None</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/Rashika40">Rashika40</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/None">None</a><br> <small> Northeastern University </small></p></li>
<li><p><strong>Jaclyn Taroni</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0003-4734-4508">0000-0003-4734-4508</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/jaclyn-taroni">jaclyn-taroni</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/jaclyn_taroni">jaclyn_taroni</a><br> <small> Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation </small></p></li>
<li><p><strong>Yihsuan Tsai</strong><br><br> <small> University of North Carolina at Chapel Hill </small></p></li>
<li><p><strong>Robin van der Lee</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0001-7391-9438">0000-0001-7391-9438</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/robinvanderlee">robinvanderlee</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/robinvdlee">robinvdlee</a><br> <small> The University of British Columbia, Vancouver, Canada; BC Children’s Hospital Research Institute; Centre for Molecular Medicine and Therapeutics </small></p></li>
<li><p><strong>Ryan Whaley</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-8810-3893">0000-0002-8810-3893</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/whaleyr">whaleyr</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/ryanatpharmgkb">ryanatpharmgkb</a><br> <small> Stanford University </small></p></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>Manubot is an open source tool for writing manuscripts on GitHub in markdown format. Manubot applies the git-based software workflow to scholarly writing, enabling enhanced transparency, collaboration, automation, and reproducibility.</p>
<p>This manuscript is the result of a <em>special working group</em> at the 2019 <a href="https://psb.stanford.edu/">Pacific Symposium on Biocomputing</a> that will introduce attendees to collaborative writing with Manubot. Each conference attendee is invited to write a small blurb on themselves and their research, by submitting a pull request to the manuscript repository at <a href="https://github.com/dhimmel/psb-manuscript" class="uri">https://github.com/dhimmel/psb-manuscript</a>.</p>
<p>The working group also <a href="https://slides.com/dhimmel/psb">covers</a> how to write your next manuscript <a href="https://manubot.org">using Manubot</a> and what features of Manubot can help biomedical researchers document and publish their computational research. For example, Manubot enables citation by persistent identifier to automate bibliographic metadata retrieval and formatting as well as allowing templating so results can be directly inserted from the analyses that produced them.</p>
<h2 id="methods" class="page_break_before">Methods</h2>
<p>In this section, PSB 2019 attendees are asked to contribute a bit about themselves and their research. As part of the <a href="http://web.archive.org/web/20190103203407/https://psb.stanford.edu/working%20group/">special working group</a>, we thought this would be a helpful activity to introduce biocomputational scientists to writing with Manubot. For inspiration, here are some prompts:</p>
<ul>
<li>Introduce yourself briefly.</li>
<li>What do you research? Include any relevant links to your lab or personal website.</li>
<li>What is your favorite study from your career or what study was your biggest discovery?</li>
<li>What was your first scholarly publication?</li>
<li>Add a code snippet or mathematical equation.</li>
<li>Add a figure with a caption. This could be a picture of you in Hawaii or a figure from your previous work if the license is permissive enough to allow reuse.</li>
</ul>
<p>Self-citations are explicitly encouraged, since one goal of this activity is to introduce attendees to the concept of <a href="https://github.com/dhimmel/psb-manuscript/blob/master/USAGE.md#citations">citation by persistent identifier</a>. By having attendees cite their existing works, we hope to show researchers how references can be created from just persistent identifiers, and how this can assist with collaborative and transparent authoring.</p>
<p>The <a href="https://github.com/dhimmel/psb-manuscript/tree/master/content">markdown manuscript source</a> has a section for each PSB 2019 attendee, generated from the online <a href="https://github.com/dhimmel/psb-manuscript/blob/master/attendees/attendees.pdf">attendee list</a>. Names are ordered alphabetically by last name. If you’d like to contribute, but are not already in the list, please insert your section at the appropriate alphabetical location.</p>
<p>For questions on how to use Manubot, see the <a href="https://github.com/dhimmel/psb-manuscript/blob/master/USAGE.md">usage documentation</a>. More information on the tool and its inception is available in the project manuscript <span class="citation" data-cites="1GGGHdsew">[<a href="#ref-1GGGHdsew">1</a>]</span>.</p>
<h2 id="attendees" class="page_break_before">Attendees</h2>
<!-- Max Alekseyev -->
<!-- Russ Altman -->
<!-- Kiyoshi Asai -->
<!-- Folkert Asselbergs -->
<!-- Ho Bae -->
<!-- Brett Beaulieu-Jones -->
<!-- George Bebis -->
<!-- Joanne Berghout -->
<!-- John Black -->
<!-- Carly Bobak -->
<!-- Mary Boland -->
<!-- Philip Bourne -->
<!-- Soline Boussard -->
<!-- Steven Brenner -->
<!-- Soren Brunak -->
<!-- Martha Bulyk -->
<!-- William Bush -->
<!-- J. Brian Byrd -->
<h3 id="j.-brian-byrd">J. Brian Byrd</h3>
<p>I’m a physician-scientist at the University of Michigan. My laboratory focuses on identifying novel biomarkers for a clinically important subtype of high blood pressure, called primary aldosteronism. Our principal interest is in detecting the transcriptional activity of the mineralocorticoid receptor <span class="citation" data-cites="GGRHirwN">[<a href="#ref-GGRHirwN">2</a>]</span>.</p>
<!-- Bin Chen -->
<!-- Rachel Chen -->
<!-- Yong Chen -->
<!-- Jane Chiang -->
<!-- Jongmun Choi -->
<!-- DongWon Choo -->
<!-- Christopher Chute -->
<!-- Leonardo Collado Torres -->
<!-- Christian Darabos -->
<!-- Supriyo De -->
<!-- Jessica De Freitas -->
<!-- Scott Delp -->
<!-- Emek Demir -->
<!-- Daisy Yi Ding -->
<!-- Valentin Dinu -->
<!-- Megan Doerr -->
<!-- Ernst Dow -->
<!-- Sorin Draghici -->
<!-- A Keith Dunker -->
<!-- Claire Duvallet -->
<!-- Prashant Emani -->
<!-- Peter Embi -->
<!-- Barbara Engelhardt -->
<!-- Yayin Fang -->
<!-- Kimberley Ferguson -->
<!-- James Foster -->
<!-- Verena Friedl -->
<!-- Weixuan Fu -->
<h3 id="weixuan-fu">Weixuan Fu</h3>
<p>Aloha, I’m in the <a href="http://upibi.org/">Institute for Biomedical Informatics (IBI)</a> at the University of Pennsylvania and the developer of <a href="https://github.com/EpistasisLab/tpot">TPOT</a> and PennAI <span class="citation" data-cites="MARd00zz">[<a href="#ref-MARd00zz">3</a>]</span>.</p>
<p>My main interest of research is developing automated machine learning tools for the analysis of large scale biomedical/sequencing data. Besides that, I am working on optimizing analysis pipeline of predicting neoantigen specifically presented in tumor cells using DNA and RNA sequencing data, for designing personalized neoantigen vaccines in cancer immunotherapies.</p>
<!-- Tetsu Furukawa -->
<!-- Lana Garmire -->
<!-- Maxwell Gold -->
<!-- Graciela Gonzalez Hernandez -->
<!-- Raluca Gordan -->
<!-- Max Gordon -->
<!-- Kiley Graim -->
<!-- Casey Greene -->
<h3 id="casey-greene">Casey Greene</h3>
<p>I run an integrative genomics research lab at the University of of Pennsylvania, and I direct the Childhood Cancer Data Lab for Alex’s Lemonade Stand Foundation. The lab at Penn develops methods to integrate large-scale public datasets, primarily from transcriptomic assays, and applies these methods to a broad set of biological questions. We’ve studied numerous systems, and we currently have active research projects in the application areas of microbial systems <span class="citation" data-cites="1CFhfCyWN Hlprh8TG">[<a href="#ref-1CFhfCyWN">4</a>,<a href="#ref-Hlprh8TG">5</a>]</span>, cancers <span class="citation" data-cites="7V2oCvtf 17XHtmqPY X6oQhIf8 11ZPgRuLk">[<a href="#ref-7V2oCvtf">6</a>,<a href="#ref-17XHtmqPY">7</a>,<a href="#ref-X6oQhIf8">8</a>,<a href="#ref-11ZPgRuLk">9</a>]</span>, and rare diseases <span class="citation" data-cites="1GtRgPRxn">[<a href="#ref-1GtRgPRxn">10</a>]</span>. At this PSB, a postdoc from the group will present a paper describing Continental Breakfast Included (CBI) effect in the final talk of the final session of this year’s meeting <span class="citation" data-cites="5CsWRjfp">[<a href="#ref-5CsWRjfp">11</a>]</span>.</p>
<p>I’m also interested in technologies that enhance the future of scientific communication. Our lab has studied Sci-Hub <span class="citation" data-cites="IhliSZDo">[<a href="#ref-IhliSZDo">12</a>]</span>. We’ve led a large collaborative review of deep learning in biology and medicine <span class="citation" data-cites="PZMP42Ak">[<a href="#ref-PZMP42Ak">13</a>]</span>. Members of the lab have developed tools like manubot <span class="citation" data-cites="1GGGHdsew">[<a href="#ref-1GGGHdsew">1</a>]</span>, which you are using now. More publications are available on our <a href="http://www.greenelab.com/publications">lab website</a>.</p>
<!-- Peyton Greenside -->
<!-- Gamze Gursoy -->
<!-- Jonathan Haines -->
<!-- Greg Hampikian -->
<!-- Jiali Han -->
<!-- Wontack Han -->
<!-- David Haussler -->
<!-- Tina Hernandez-Boussard -->
<!-- Daniel Himmelstein -->
<h3 id="daniel-himmelstein">Daniel Himmelstein</h3>
<p>Greetings, I’m in the <a href="http://www.greenelab.com/">Greene Lab</a> at the University of Pennsylvania and am the lead developer of the Manubot project. 2019 is my first PSB and I’m exciting to backpack around the Big Island following the conference.</p>
<p>My main area of research is integrating biomedical knowledge using hetnets <span class="citation" data-cites="XJuQlZl1 O21tn8vf">[<a href="#ref-XJuQlZl1">14</a>,<a href="#ref-O21tn8vf">15</a>]</span>. However, I’ve also studied Sci-Hub, finding that it provides access to nearly all paywalled scholarly literature <span class="citation" data-cites="vkig9eAb">[<a href="#ref-vkig9eAb">16</a>]</span>. Perhaps my biggest discovery was observing an epidemiological association that higher elevation counties have lower rates of lung cancer, suggesting that oxygen is an inhaled carcinogen (Figure <a href="#fig:elevcan">1</a>) <span class="citation" data-cites="17wiYglmF 8YjPCnYe">[<a href="#ref-17wiYglmF">17</a>,<a href="#ref-8YjPCnYe">18</a>]</span>.</p>
<figure>
<img src="https://github.com/dhimmel/elevcan/raw/7aed9f29d2371eb4918f337a138608e6b6d9e311/manual/figures/peerj/Figure_4.png" alt="Figure 1: The association between elevation and lung cancer across Western U.S. counties. This figure is reused from here under its CC BY 4.0 License." id="fig:elevcan" style="width:100.0%" /><figcaption><span>Figure 1:</span> The association between elevation and lung cancer across Western U.S. counties. This figure is reused from <a href="https://doi.org/10.7717/peerj.705/fig-4">here</a> under its CC BY 4.0 License.</figcaption>
</figure>
<p>I haven’t done much text mining, but I did enjoy extracting attendee names for PSB from the online PDF. Converting the PDF to text in Python was <a href="https://github.com/dhimmel/psb-manuscript/blob/15babecdf2a915f88088703e23a61e34e1294b1f/attendees/attendees.ipynb">as easy as</a>:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode python"><code class="sourceCode python"><a class="sourceLine" id="cb1-1" title="1"><span class="co"># https://stackoverflow.com/a/48673754</span></a>
<a class="sourceLine" id="cb1-2" title="2"><span class="im">import</span> tika.parser</a>
<a class="sourceLine" id="cb1-3" title="3">parsed <span class="op">=</span> tika.parser.from_file(<span class="st">&#39;attendees.pdf&#39;</span>)</a>
<a class="sourceLine" id="cb1-4" title="4">text <span class="op">=</span> parsed[<span class="st">&quot;content&quot;</span>]</a></code></pre></div>
<!-- Michael Hoffman -->
<!-- John Holmes -->
<!-- Rachel Hovde -->
<!-- Qiwen Hu -->
<h3 id="qiwen-hu">Qiwen Hu</h3>
<p>I’m a postdoc from <a href="http://www.greenelab.com/">Greene Lab</a> at the University of Pennsylvania. My research focus on integrating different types of high-throughput sequencing data to find meaningful biological signals behind it. I developed machine learning and statistical approaches to identify regulatory elements that affect transcription and translation. I also developed machine learning-based methods to extract regulatory signals from addicted brain <span class="citation" data-cites="1m3Egymh">[<a href="#ref-1m3Egymh">19</a>]</span>, developmental tissues <span class="citation" data-cites="M0crDyWe">[<a href="#ref-M0crDyWe">20</a>]</span> and cell-type signals from single-cell datasets.</p>
<p>This year at PSB, I will present our findings for analyzing single-cell data based on deep variation auto-encoders <span class="citation" data-cites="5CsWRjfp">[<a href="#ref-5CsWRjfp">11</a>]</span>.</p>
<!-- Zhiyue Hu -->
<!-- Lawrence Hunter -->
<h3 id="lawrence-hunter">Lawrence Hunter</h3>
<p>I’m a cofounder of the <a href="http://psb.stanford.edu">PSB conference</a>, and a professor at the University of Colorado School of Medicine. You can find information about my lab at <a href="http://compbio.ucdenver.edu/Hunter" class="uri">http://compbio.ucdenver.edu/Hunter</a>. One of my early papers is <span class="citation" data-cites="mFhozBp8"><a href="#ref-mFhozBp8">21</a></span>.</p>
<!-- Shantanu Jain -->
<h3 id="shantanu-jain">Shantanu Jain</h3>
<p>Hi all, I am very excited to be here attending PSB. I am a research scientist at Northeastern University. I am broadly interested in Machine learning methods. During my Ph.D., I worked on positive unlabeled learning. I am most proud about my research on non parametric estimation of class priors from positive and unlabeled data <span class="citation" data-cites="BYxifaL7">[<a href="#ref-BYxifaL7">22</a>]</span>. I have started learning about Causal Inference lately and I am interested in applying it to Biological datasets.</p>
<!-- Yuexu Jiang -->
<!-- Kipp Johnson -->
<!-- Kory Johnson -->
<!-- Indika Kahanda -->
<!-- Laurynas Kalesinskas -->
<!-- Michael Keiser -->
<!-- Andreas Keller -->
<!-- Colleen Kenost -->
<!-- Aly Khan -->
<!-- Dokyoon Kim -->
<!-- Jeremie Kim -->
<!-- Teri Klein -->
<!-- Kord Kober -->
<!-- Milica Krunic -->
<!-- Adam Kurkiewicz -->
<h3 id="adam-kurkiewicz">Adam Kurkiewicz</h3>
<p>I’m interested in building a tool to do SNP calling from single cell RNASeq data. This has been tried before by various groups, e.g. check out the honeyBADGER paper <span class="citation" data-cites="1elWGdf0">[<a href="#ref-1elWGdf0">23</a>]</span>, but ultimately none of the approaches were successful. I have a few ideas on how to make progress — give me a shout if you’d like to discuss!</p>
<!-- Nicholas Larson -->
<!-- Janet Layne -->
<!-- Trang Le -->
<h3 id="trang-le">Trang Le</h3>
<p>Hello from the <a href="http://epistasis.org/">Moore lab</a> at the University of Pennsylvania! I’m <a href="http://lelaboratoire.github.io">a mathematician</a> who’s currently excited about automated machine learning.</p>
<p>Here goes the self-citations:</p>
<ul>
<li>My own favorite study: Generalization of the Fermi Pseudopotential <span class="citation" data-cites="10GH4uYUR">[<a href="#ref-10GH4uYUR">24</a>]</span> - a piece of mathematical physics work I got to do when procrastinating writing my dissertation.</li>
<li>My first (first-author) scholarly publication: Differential privacy-based evaporative cooling feature selection and classification with relief-F and random forests <span class="citation" data-cites="hUoMaAJ6">[<a href="#ref-hUoMaAJ6">25</a>]</span>. Check out the Github repo for this study <a href="https://github.com/insilico/privateEC">here</a>.</li>
<li><p>Code snippet I’m most proud of:</p>
<pre><code>M = dec2bin(0:2^(n*n)-1,n*n)</code></pre>
I will be impressed if you could tell what the language is. This is my answer to a question on <a href="https://math.stackexchange.com/questions/1943862/matlab-code-for-an-array-consisting-of-matrices/1943923#1943923">Math StackExchange</a>.</li>
<li>I have too many favorite mathematical equations, but here’s one: <span class="math display">\[a^p \equiv a \mod p\]</span> Anyone recognize this theorem?</li>
<li><p>And a figure with a caption:</p>
<figure>
<img src="https://s3.amazonaws.com/media-p.slid.es/uploads/909204/images/5547949/fig2.png" alt="Figure 2: Participants tended to have older brain-predicted age when given placebo." id="fig:ibuBrainAGE" style="width:80.0%" /><figcaption><span>Figure 2:</span> Participants tended to have older brain-predicted age when given placebo.</figcaption>
</figure></li>
</ul>
<p>This is an improved version of my main figure in this interesting study <span class="citation" data-cites="ih7MBGjv">[<a href="#ref-ih7MBGjv">26</a>]</span>.</p>
<!-- Jinhee Lee -->
<!-- Nicholas Lee -->
<!-- DoHyeon Lee -->
<!-- Binglan Li -->
<h3 id="binglan-li">Binglan Li</h3>
<p>Greeting from the <a href="http://ritchielab.org/">Ritchie Lab</a> at the University of Pennsylvania. I am a third year graduate student in the Genomics and Computational Biology programe and interested in prioritization of drug response-related gene via data integration approaches.</p>
<p>I am still on the early part of research journey. But I would love to share my latest work published in the PSB 2019 proceedings 2019.</p>
<ul>
<li><p>Influence of tissue context on gene prioritization for predicted transcriptome-wide association studies <span class="citation" data-cites="9B8DWkq3">[<a href="#ref-9B8DWkq3">27</a>]</span>.</p></li>
<li>Code snippet I’m most proud of:</li>
</ul>
<pre><code>############################################
## Menu
## 1. Food Preparation
## 1.1. Load Necessary Libraries and Scripts
## 1.2. Define Parameters
## 2. Appetizers
## 2.1. Data Simulation
## 2.2. eQTL Detection
## 3. Entree/Main Course
## 3.1. Run single-tissue TWAS
## 3.2. Run integrative TWAS
## 3.3. Evaluate Power and Type I Error Rate of TWAS Results
## 4. Dessert
## 4.1. None. Sorry this is a healthy (aka anti-sweet) restaurant.
############################################

## actual code
set.seed(random_seed, kind = &quot;L&#39;Ecuyer-CMRG&quot;)</code></pre>
<ul>
<li>Here is a plot about minor allele frequency of the eQTLs in the GTEx v7 whole blood tissue. Please pretend that you do see a title in the figure.</li>
</ul>
<figure>
<img src="https://user-images.githubusercontent.com/1117703/50731107-cd16b380-1100-11e9-930c-3a23cb4a0ec9.png" alt="Figure 3: Minor allele frequency of eQTLs in the GTEx v7 whole blood tissue." id="fig:binglan" style="width:50.0%" /><figcaption><span>Figure 3:</span> Minor allele frequency of eQTLs in the GTEx v7 whole blood tissue.</figcaption>
</figure>
<!-- Haiquan Li -->
<!-- Hua Li -->
<!-- Jianrong Li -->
<!-- Kevin Li -->
<!-- Sooyeon Lim -->
<!-- Hongfang Liu -->
<!-- Zheng Liu -->
<!-- Xueqing Lu -->
<!-- Zhiyong Lu -->
<!-- Yves Lussier -->
<!-- Arjun Magge -->
<!-- Lara Mangravite -->
<!-- Serghei Mangul -->
<!-- Gabor Marth -->
<!-- Rachel Marty -->
<!-- Wouter Meuleman -->
<!-- Jason Miller -->
<h3 id="jason-e.-miller">Jason E. Miller</h3>
<p>Hi, I’m a postdoctoral fellow from the <a href="https://ritchielab.org">Ritchie lab</a> at the University of Pennsylvania.</p>
<p>I’m currently focussed on identifying how genetic variation leads to Alzheimer’s disease through perturbation of gene regulatory mechanisms.</p>
<p>My favorite study from my career identified specific types of codon bias among synonymous variants, such as those related to codon optimality and frequency, that are associated with an Alzheimer’s disease imaging endophenotype <span class="citation" data-cites="czfw16Ig">[<a href="#ref-czfw16Ig">28</a>]</span>.</p>
<p>If you are interested, you can check out my github page <a href="https://github.com/git-jemiller">here</a>.</p>
<!-- Stephen Montgomery -->
<!-- Abigail Moore -->
<!-- Jason Moore -->
<!-- Alexander Morgan -->
<!-- Sorana Morrissy -->
<!-- Meghan Muse -->
<!-- Rikke Nielsen -->
<!-- William Noble -->
<!-- Soichi Ogishima -->
<!-- Lawrence Oloff -->
<!-- Zhengqing Ouyang -->
<!-- Philip Payne -->
<!-- Matteo Pellegrini -->
<!-- Luca Pinello -->
<h3 id="luca-pinello">Luca Pinello</h3>
<p>Aloah from the <a href="http://pinellolab.org/">Pinello Lab!</a></p>
<p>I am a computational biologist studying the role of chromatin structure/dynamics and non-coding regions including enhancers, promoters, insulators and their role in gene regulation. The mission of my lab is the integration of omics data to explore and better understand the functional mechanisms of the non-coding genome and to provide accessible tools for the community to accelerate discovery in this field. The long-term goal of my research is to develop innovative computational approaches and to use cutting-edge experimental assays, such as single cell and genome editing, to systematically analyze sources of genetic and epigenetic variation that affect gene regulation in different human traits and diseases. I believe this will further our understanding of disease etiology involving these poorly characterized regions and will provide a foundation for the development of new drugs and more targeted treatments.</p>
<p>I am excited to share during the workshop <a href="http://lussierlab.net/pacific-symposium-2019/index.html">Reading between the genes: Interpreting noncoding DNA in high throughput</a> a new computational methods we have recently developed to analyze CRISPR tiling screen called CRISPR-SURF. You can read more on the manuscript that was recently published in Nature Methods <span class="citation" data-cites="WXZIineW">[<a href="#ref-WXZIineW">29</a>]</span>.</p>
<!-- Niranjani Prasad -->
<!-- Blake Pyman -->
<!-- Tielin Qin -->
<!-- Aaron Quinlan -->
<!-- Predrag Radivojac -->
<!-- Rashika Ramola -->
<h3 id="rashika-ramola">Rashika Ramola</h3>
<p>Hi I am Rashika Ramola. I am a PhD student at Northeastern University. This is my first PSB. I like computational biology, and I am excited to be here.</p>
<p>My first paper studies some performance measures (accuracy, balanced accuracy, f-measure and Matthews Correlation Coefficient) in positive-unlabeled learning <span class="citation" data-cites="10OhwB9If">[<a href="#ref-10OhwB9If">30</a>]</span>. In this work, we demonstrate how performance measure can be inaccurate in positive unlabeled setting, and then we introduce correction measures.</p>
<p>I am including an important formula from the abovementioned mauscript:</p>
<p><span class="math display">\[
\mcc=\frac{1}{\mpl-\mpu}\sqrt{\frac{\mpt\paren*{1-\mpt}}{\propL(1-\propL)}}\cdot \mccPU
\]</span></p>
<p>It shows that Matthews correlation coefficient (MCC) is directly proportional to its equivalent in positive unlabeled setting. Thus, MCC is a well behaved performance measure.</p>
<p>Here is a beautiful aerial shot of <a href="https://www.myhawaii.com.au/wp-content/uploads/sites/13/2018/08/Hawaii-Landscape-Copy.jpg">Hawaii</a>.</p>
<!-- Jayamary Divya Ravichandar -->
<!-- Derek Reiman -->
<!-- Haluk Resat -->
<!-- Marylyn Ritchie -->
<!-- David Rocke -->
<!-- Pratyaydipta Rudra -->
<!-- Indra Sarkar -->
<!-- Sebastian Schaaf -->
<!-- Ralph Schlapbach -->
<!-- Christine Scholberg -->
<!-- Ashwini Sehgal -->
<!-- Martin Seneviratne -->
<!-- Matthew Settles -->
<!-- James Sikela -->
<!-- Sean Simmons -->
<!-- Marina Sirota -->
<!-- Giltae Song -->
<!-- Paul Spellman -->
<!-- Arunima Srivastava -->
<!-- Josh Stuart -->
<!-- Yoichi Takenaka -->
<!-- Eric Talevich -->
<!-- Haixu Tang -->
<!-- Lin Tang -->
<!-- Yosuke Tanigawa -->
<!-- Cui Tao -->
<!-- Jaclyn Taroni -->
<h3 id="jaclyn-taroni">Jaclyn Taroni</h3>
<p>I’m a data scientist at the <a href="https://www.ccdatalab.org">Childhood Cancer Data Lab</a> (CCDL), an initiative of <a href="https://www.alexslemonade.org">Alex’s Lemonade Stand Foundation</a>. I’m interested in how diverse collections of publicly available transcriptomic data can help us learn about the biology of rare diseases. As a graduate student, I studied systemic sclerosis <span class="citation" data-cites="1Av3x1pnk">[<a href="#ref-1Av3x1pnk">31</a>]</span>. In the <a href="https://healthlanguageprocessing.org/psb-2019-tm-ml-pm-workshop/">PSB 2019 Text Mining and Machine Learning for Precision Medicine Workshop</a>, I’ll present our MultiPLIER project <span class="citation" data-cites="1GtRgPRxn">[<a href="#ref-1GtRgPRxn">10</a>]</span>. With the CCDL, I’ve been working on <a href="https://www.refine.bio">refine.bio</a>, a project for uniformly processing transcriptomic data from multiple species.</p>
<!-- Gunjan Thakur -->
<!-- Robert Thurman -->
<!-- Nathan Tintle -->
<!-- Yihsuan Tsai -->
<h3 id="yihsuan-tsai-shannon">Yihsuan Tsai (Shannon)</h3>
<p>This is Shannon from UNC at Chapel Hill. I’m a bioinformatics scientis at <a href="https://lbc.unc.edu/">UNC lineberger cancer center</a>. My recent research project could be found at PSB poster section #69. It’s about using methylation data to predict tumor infiltrating lymphocytes, which is highly correlated with patient survival in Melanoma.</p>
<p>Here are some of my publications:</p>
<ol type="1">
<li>Meta-analysis of airway epithelium gene expression in asthma <span class="citation" data-cites="r7HvVKdn">[<a href="#ref-r7HvVKdn">32</a>]</span>.</li>
<li>Identification of a robust methylation classifier for cutaneous melanoma diagnosis <span class="citation" data-cites="yKr4neGw">[<a href="#ref-yKr4neGw">33</a>]</span>.</li>
<li>Transcriptome-wide identification and study of cancer-specific splicing events across multiple tumors <span class="citation" data-cites="9sfG2U7T">[<a href="#ref-9sfG2U7T">34</a>]</span>.</li>
<li>Prevalent RNA recognition motif duplication in the human genome <span class="citation" data-cites="jk5tACP8">[<a href="#ref-jk5tACP8">35</a>]</span>.</li>
</ol>
<!-- Brian Yik Tak Tsui -->
<!-- Anders Ulrik Eliasen -->
<!-- Paul Utz -->
<!-- Robin van der Lee -->
<h3 id="robin-van-der-lee">Robin van der Lee</h3>
<p>Hi! I’m a post-doc with Wyeth Wasserman at UBC, Vancouver, Canada. Info about the lab can be found at <a href="http://www.cisreg.ca" class="uri">http://www.cisreg.ca</a> and <a href="https://github.com/wassermanlab" class="uri">https://github.com/wassermanlab</a>.</p>
<p>My PhD work was on integrative omics to discover genes involved in immunity <span class="citation" data-cites="6zDUsoSA">[<a href="#ref-6zDUsoSA">36</a>]</span>. I also did some work on comparative genomics of primate genomes, finding that rapidly evolving genes are predictive of virus-human interactions <span class="citation" data-cites="Myj0Ouv5">[<a href="#ref-Myj0Ouv5">37</a>]</span>.</p>
<p>In my post-doc work, I am developing methods for interpreting regulatory genomic variants based on alterations to transcription factor binding motifs. Some of that work is on <strong>poster 71</strong>, which I will present on Saturday 5 January 2018 at the PSB meeting.</p>
<figure>
<img src="images/Robin-van-der-Lee_Poster_Pacific_Symposium_on_Biocomputing_PSB_2019__MANTA-RAE_poster_header.jpg" alt="Figure 4: This is the header of the poster I’ll present here under its CC BY 4.0 License." id="fig:vanderlee" style="width:100.0%" /><figcaption><span>Figure 4:</span> This is the header of the poster I’ll present here under its CC BY 4.0 License.</figcaption>
</figure>
<!-- Maya Varma -->
<!-- Anurag Verma -->
<!-- Shefali Verma -->
<!-- Yogasudha Veturi -->
<!-- Francesca Vitali -->
<!-- Dennis Wall -->
<!-- Duolin Wang -->
<!-- Haohan Wang -->
<!-- Jing Wang -->
<!-- Tongxin Wang -->
<!-- Yong Wang -->
<!-- Joanne Watson -->
<!-- Ryan Whaley -->
<h3 id="ryan-whaley">Ryan Whaley</h3>
<p>Hi, I’m Ryan and I’m one of the technical leads for <a href="https://www.pharmgkb.org">PharmGKB</a>. I’m also helping to run the A/V desk during this presentation.</p>
<p>I’m trained in software development and started by career as a DBA. Over the past decade I’ve switched to Java and then web application development. I’ve contributed to PharmGKB <span class="citation" data-cites="3NKcTIrb">[<a href="#ref-3NKcTIrb">38</a>]</span>, CPIC <span class="citation" data-cites="1EYG4hAqR">[<a href="#ref-1EYG4hAqR">39</a>]</span>, and other PGx consortia.</p>
<!-- Nicholas Wheeler -->
<!-- Michelle Whirl-Carrillo -->
<!-- Scott Williams -->
<!-- John Witte -->
<!-- Yuanxin Xi -->
<!-- Madelyne Xiao -->
<!-- Yuzhen Ye -->
<!-- Shuxing Zhang -->
<!-- Xinyuan Zhang -->
<!-- Yuping Zhang -->
<!-- Zemin Zhang -->
<!-- the end -->
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-1GGGHdsew">
<p>1. <strong>Open collaborative writing with Manubot</strong><br />
Daniel S. Himmelstein, David R. Slochower, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>Manubot Preprint</em> (2018-12-31) <a href="https://greenelab.github.io/meta-review/">https://greenelab.github.io/meta-review/</a></p>
</div>
<div id="ref-GGRHirwN">
<p>2. <strong>Human Urinary mRNA as a Biomarker of Cardiovascular Disease.</strong><br />
Brian G Bazzell, William E Rainey, Richard J Auchus, Davide Zocco, Marco Bruttini, Scott L Hummel, James Brian Byrd<br />
<em>Circulation. Genomic and precision medicine</em> (2018-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30354328">https://www.ncbi.nlm.nih.gov/pubmed/30354328</a><br />
DOI: <a href="https://doi.org/10.1161/circgen.118.002213">10.1161/circgen.118.002213</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30354328">30354328</a></p>
</div>
<div id="ref-MARd00zz">
<p>3. <strong>A System for Accessible Artificial Intelligence</strong><br />
Randal S. Olson, Moshe Sipper, William La Cava, Sharon Tartarone, Steven Vitale, Weixuan Fu, Patryk Orzechowski, Ryan J. Urbanowicz, John H. Holmes, Jason H. Moore<br />
<em>Genetic Programming Theory and Practice XV</em> (2018) <a href="https://doi.org/gfsptm">https://doi.org/gfsptm</a><br />
DOI: <a href="https://doi.org/10.1007/978-3-319-90512-9_8">10.1007/978-3-319-90512-9_8</a></p>
</div>
<div id="ref-1CFhfCyWN">
<p>4. <strong>ADAGE-Based Integration of Publicly Available Pseudomonas aeruginosa Gene Expression Data with Denoising Autoencoders Illuminates Microbe-Host Interactions</strong><br />
Jie Tan, John H. Hammond, Deborah A. Hogan, Casey S. Greene<br />
<em>mSystems</em> (2016-01-19) <a href="https://doi.org/gcgmbq">https://doi.org/gcgmbq</a><br />
DOI: <a href="https://doi.org/10.1128/msystems.00025-15">10.1128/msystems.00025-15</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27822512">27822512</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5069748">PMC5069748</a></p>
</div>
<div id="ref-Hlprh8TG">
<p>5. <strong>Unsupervised Extraction of Stable Expression Signatures from Public Compendia with an Ensemble of Neural Networks</strong><br />
Jie Tan, Georgia Doing, Kimberley A. Lewis, Courtney E. Price, Kathleen M. Chen, Kyle C. Cady, Barret Perchuk, Michael T. Laub, Deborah A. Hogan, Casey S. Greene<br />
<em>Cell Systems</em> (2017-07) <a href="https://doi.org/gcw9f4">https://doi.org/gcw9f4</a><br />
DOI: <a href="https://doi.org/10.1016/j.cels.2017.06.003">10.1016/j.cels.2017.06.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28711280">28711280</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5532071">PMC5532071</a></p>
</div>
<div id="ref-7V2oCvtf">
<p>6. <strong>Extracting a biologically relevant latent space from cancer transcriptomes with variational autoencoders</strong><br />
Gregory P. Way, Casey S. Greene<br />
<em>Biocomputing 2018</em> (2017-11-17) <a href="https://doi.org/gfspsd">https://doi.org/gfspsd</a><br />
DOI: <a href="https://doi.org/10.1142/9789813235533_0008">10.1142/9789813235533_0008</a></p>
</div>
<div id="ref-17XHtmqPY">
<p>7. <strong>Machine Learning Detects Pan-cancer Ras Pathway Activation in The Cancer Genome Atlas</strong><br />
Gregory P. Way, Francisco Sanchez-Vega, Konnor La, Joshua Armenia, Walid K. Chatila, Augustin Luna, Chris Sander, Andrew D. Cherniack, Marco Mina, Giovanni Ciriello, … Armaz Mariamidze<br />
<em>Cell Reports</em> (2018-04) <a href="https://doi.org/gfspsb">https://doi.org/gfspsb</a><br />
DOI: <a href="https://doi.org/10.1016/j.celrep.2018.03.046">10.1016/j.celrep.2018.03.046</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29617658">29617658</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918694">PMC5918694</a></p>
</div>
<div id="ref-X6oQhIf8">
<p>8. <strong>Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome Atlas</strong><br />
Theo A. Knijnenburg, Linghua Wang, Michael T. Zimmermann, Nyasha Chambwe, Galen F. Gao, Andrew D. Cherniack, Huihui Fan, Hui Shen, Gregory P. Way, Casey S. Greene, … Armaz Mariamidze<br />
<em>Cell Reports</em> (2018-04) <a href="https://doi.org/gfspsc">https://doi.org/gfspsc</a><br />
DOI: <a href="https://doi.org/10.1016/j.celrep.2018.03.076">10.1016/j.celrep.2018.03.076</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29617664">29617664</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5961503">PMC5961503</a></p>
</div>
<div id="ref-11ZPgRuLk">
<p>9. <strong>Oncogenic Signaling Pathways in The Cancer Genome Atlas</strong><br />
Francisco Sanchez-Vega, Marco Mina, Joshua Armenia, Walid K. Chatila, Augustin Luna, Konnor C. La, Sofia Dimitriadoy, David L. Liu, Havish S. Kantheti, Sadegh Saghafinia, … Armaz Mariamidze<br />
<em>Cell</em> (2018-04) <a href="https://doi.org/gc7r9b">https://doi.org/gc7r9b</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2018.03.035">10.1016/j.cell.2018.03.035</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29625050">29625050</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6070353">PMC6070353</a></p>
</div>
<div id="ref-1GtRgPRxn">
<p>10. <strong>MultiPLIER: a transfer learning framework reveals systemic features of rare autoimmune disease</strong><br />
Jaclyn N Taroni, Peter C Grayson, Qiwen Hu, Sean Eddy, Matthias Kretzler, Peter A Merkel, Casey S Greene<br />
<em>Cold Spring Harbor Laboratory</em> (2018-08-20) <a href="https://doi.org/gfc9bb">https://doi.org/gfc9bb</a><br />
DOI: <a href="https://doi.org/10.1101/395947">10.1101/395947</a></p>
</div>
<div id="ref-5CsWRjfp">
<p>11. <strong>Parameter tuning is a key part of dimensionality reduction via deep variational autoencoders for single cell RNA transcriptomics</strong><br />
Qiwen Hu, Casey S Greene<br />
<em>Cold Spring Harbor Laboratory</em> (2018-08-05) <a href="https://doi.org/gdxxjf">https://doi.org/gdxxjf</a><br />
DOI: <a href="https://doi.org/10.1101/385534">10.1101/385534</a></p>
</div>
<div id="ref-IhliSZDo">
<p>12. <strong>Sci-Hub provides access to nearly all scholarly literature</strong><br />
Daniel S Himmelstein, Ariel Rodriguez Romero, Jacob G Levernier, Thomas Anthony Munro, Stephen Reid McLaughlin, Bastian Greshake Tzovaras, Casey S Greene<br />
<em>eLife</em> (2018-03-01) <a href="https://doi.org/ckcj">https://doi.org/ckcj</a><br />
DOI: <a href="https://doi.org/10.7554/elife.32822">10.7554/elife.32822</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29424689">29424689</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5832410">PMC5832410</a></p>
</div>
<div id="ref-PZMP42Ak">
<p>13. <strong>Opportunities and obstacles for deep learning in biology and medicine</strong><br />
Travers Ching, Daniel S. Himmelstein, Brett K. Beaulieu-Jones, Alexandr A. Kalinin, Brian T. Do, Gregory P. Way, Enrico Ferrero, Paul-Michael Agapow, Michael Zietz, Michael M. Hoffman, … Casey S. Greene<br />
<em>Journal of The Royal Society Interface</em> (2018-04) <a href="https://doi.org/gddkhn">https://doi.org/gddkhn</a><br />
DOI: <a href="https://doi.org/10.1098/rsif.2017.0387">10.1098/rsif.2017.0387</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29618526">29618526</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5938574">PMC5938574</a></p>
</div>
<div id="ref-XJuQlZl1">
<p>14. <strong>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes.</strong><br />
Daniel S Himmelstein, Sergio E Baranzini<br />
<em>PLoS computational biology</em> (2015-07-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26158728">https://www.ncbi.nlm.nih.gov/pubmed/26158728</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1004259">10.1371/journal.pcbi.1004259</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26158728">26158728</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4497619">PMC4497619</a></p>
</div>
<div id="ref-O21tn8vf">
<p>15. <strong>Systematic integration of biomedical knowledge prioritizes drugs for repurposing</strong><br />
Daniel Scott Himmelstein, Antoine Lizee, Christine Hessler, Leo Brueggeman, Sabrina L Chen, Dexter Hadley, Ari Green, Pouya Khankhanian, Sergio E Baranzini<br />
<em>eLife</em> (2017-09-22) <a href="https://doi.org/cdfk">https://doi.org/cdfk</a><br />
DOI: <a href="https://doi.org/10.7554/elife.26726">10.7554/elife.26726</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28936969">28936969</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640425">PMC5640425</a></p>
</div>
<div id="ref-vkig9eAb">
<p>16. <strong>Sci-Hub provides access to nearly all scholarly literature</strong><br />
Daniel S Himmelstein, Ariel Rodriguez Romero, Jacob G Levernier, Thomas Anthony Munro, Stephen Reid McLaughlin, Bastian Greshake Tzovaras, Casey S Greene<br />
<em>eLife</em> (2018-03-01) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5832410/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5832410/</a><br />
DOI: <a href="https://doi.org/10.7554/eLife.32822">10.7554/elife.32822</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29424689">29424689</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5832410">PMC5832410</a></p>
</div>
<div id="ref-17wiYglmF">
<p>17. <strong>Lung cancer incidence decreases with elevation: evidence for oxygen as an inhaled carcinogen</strong><br />
Kamen P. Simeonov, Daniel S. Himmelstein<br />
<em>PeerJ</em> (2015-01-13) <a href="https://doi.org/98p">https://doi.org/98p</a><br />
DOI: <a href="https://doi.org/10.7717/peerj.705">10.7717/peerj.705</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25648772">25648772</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4304851">PMC4304851</a></p>
</div>
<div id="ref-8YjPCnYe">
<p>18. <strong>Unraveling the Ties of Altitude, Oxygen and Lung Cancer</strong><br />
George Johnson<br />
<em>The New York Times</em> (2016-01-25) <a href="https://www.nytimes.com/2016/01/26/science/unraveling-the-ties-of-altitude-oxygen-and-lung-cancer.html">https://www.nytimes.com/2016/01/26/science/unraveling-the-ties-of-altitude-oxygen-and-lung-cancer.html</a></p>
</div>
<div id="ref-1m3Egymh">
<p>19. <strong>Histone posttranslational modifications predict specific alternative exon subtypes in mammalian brain</strong><br />
Qiwen Hu, Eun Ji Kim, Jian Feng, Gregory R. Grant, Elizabeth A. Heller<br />
<em>PLOS Computational Biology</em> (2017-06-13) <a href="https://doi.org/gbhkps">https://doi.org/gbhkps</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1005602">10.1371/journal.pcbi.1005602</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28609483">28609483</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5487056">PMC5487056</a></p>
</div>
<div id="ref-M0crDyWe">
<p>20. <strong>Specific histone modifications associate with alternative exon selection during mammalian development</strong><br />
Qiwen Hu, Casey Greene, Elizabeth Heller<br />
<em>Cold Spring Harbor Laboratory</em> (2018-07-04) <a href="https://doi.org/gfsptv">https://doi.org/gfsptv</a><br />
DOI: <a href="https://doi.org/10.1101/361816">10.1101/361816</a></p>
</div>
<div id="ref-mFhozBp8">
<p>21. <strong>EDGAR: Extraction of Drugs, Genes And Relations from the Biomedical Literature</strong><br />
Thomas C. Rindflesch, Lorraine Tanabe, John N. Weinstein, Lawrence Hunter<br />
<em>Biocomputing 2000</em> (1999-12) <a href="https://doi.org/gfsptq">https://doi.org/gfsptq</a><br />
DOI: <a href="https://doi.org/10.1142/9789814447331_0049">10.1142/9789814447331_0049</a></p>
</div>
<div id="ref-BYxifaL7">
<p>22. <strong>Nonparametric semi-supervised learning of class proportions</strong><br />
Shantanu Jain, Martha White, Michael W. Trosset, Predrag Radivojac<br />
<em>arXiv</em> (2016-01-08) <a href="https://arxiv.org/abs/1601.01944v1">https://arxiv.org/abs/1601.01944v1</a></p>
</div>
<div id="ref-1elWGdf0">
<p>23. <strong>Linking transcriptional and genetic tumor heterogeneity through allele analysis of single-cell RNA-seq data</strong><br />
Jean Fan, Hae-Ock Lee, Soohyun Lee, Da-eun Ryu, Semin Lee, Catherine Xue, Seok Jin Kim, Kihyun Kim, Nikolaos Barkas, Peter J. Park, … Peter V. Kharchenko<br />
<em>Genome Research</em> (2018-06-13) <a href="https://doi.org/gdrgwz">https://doi.org/gdrgwz</a><br />
DOI: <a href="https://doi.org/10.1101/gr.228080.117">10.1101/gr.228080.117</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29898899">29898899</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6071640">PMC6071640</a></p>
</div>
<div id="ref-10GH4uYUR">
<p>24. <strong>Generalization of the Fermi Pseudopotential</strong><br />
Trang T. Le, Zach Osman, D. K. Watson, Martin Dunn, B. A. McKinney<br />
<em>arXiv</em> (2018-06-14) <a href="https://arxiv.org/abs/1806.05726v1">https://arxiv.org/abs/1806.05726v1</a></p>
</div>
<div id="ref-hUoMaAJ6">
<p>25. <strong>Differential privacy-based evaporative cooling feature selection and classification with relief-F and random forests</strong><br />
Trang T Le, W Kyle Simmons, Masaya Misaki, Jerzy Bodurka, Bill C White, Jonathan Savitz, Brett A McKinney<br />
<em>Bioinformatics</em> (2017-05-04) <a href="https://doi.org/f96b8d">https://doi.org/f96b8d</a><br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btx298">10.1093/bioinformatics/btx298</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28472232">28472232</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5870708">PMC5870708</a></p>
</div>
<div id="ref-ih7MBGjv">
<p>26. <strong>Effect of Ibuprofen on BrainAGE: A Randomized, Placebo-Controlled, Dose-Response Exploratory Study</strong><br />
Trang T. Le, Rayus Kuplicki, Hung-Wen Yeh, Robin L. Aupperle, Sahib S. Khalsa, W. Kyle Simmons, Martin P. Paulus<br />
<em>Biological Psychiatry: Cognitive Neuroscience and Neuroimaging</em> (2018-10) <a href="https://doi.org/gfsprv">https://doi.org/gfsprv</a><br />
DOI: <a href="https://doi.org/10.1016/j.bpsc.2018.05.002">10.1016/j.bpsc.2018.05.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29941380">29941380</a></p>
</div>
<div id="ref-9B8DWkq3">
<p>27. <strong>Influence of tissue context on gene prioritization for predicted transcriptome-wide association studies</strong><br />
Binglan Li, Yogasudha Veturi, Yuki Bradford, Shefali S. Verma, Anurag Verma, Anastasia M. Lucas, David W. Haas, Marylyn D. Ritchie<br />
<em>Biocomputing 2019</em> (2018-11) <a href="https://doi.org/gfsqxj">https://doi.org/gfsqxj</a><br />
DOI: <a href="https://doi.org/10.1142/9789813279827_0027">10.1142/9789813279827_0027</a></p>
</div>
<div id="ref-czfw16Ig">
<p>28. <strong>Codon bias among synonymous rare variants is associated with Alzheimer’s disease imaging biomarker</strong><br />
Jason E Miller, Manu K Shivakumar, Shannon L Risacher, Andrew J Saykin, Seunggeun Lee, Kwangsik Nho, Dokyoon Kim<br />
<em>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</em> (2018) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5756629/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5756629/</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29218897">29218897</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5756629">PMC5756629</a></p>
</div>
<div id="ref-WXZIineW">
<p>29. <strong>CRISPR-SURF: discovering regulatory elements by deconvolution of CRISPR tiling screen data.</strong><br />
Jonathan Y Hsu, Charles P Fulco, Mitchel A Cole, Matthew C Canver, Danilo Pellin, Falak Sher, Rick Farouni, Kendell Clement, Jimmy A Guo, Luca Biasco, … Luca Pinello<br />
<em>Nature methods</em> (2018-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30504875">https://www.ncbi.nlm.nih.gov/pubmed/30504875</a><br />
DOI: <a href="https://doi.org/10.1038/s41592-018-0225-6">10.1038/s41592-018-0225-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30504875">30504875</a></p>
</div>
<div id="ref-10OhwB9If">
<p>30. <strong>Estimating classification accuracy in positive-unlabeled learning: characterization and correction strategies</strong><br />
Rashika Ramola, Shantanu Jain, Predrag Radivojac<br />
<em>Biocomputing 2019</em> (2018-11) <a href="https://doi.org/gfspvd">https://doi.org/gfspvd</a><br />
DOI: <a href="https://doi.org/10.1142/9789813279827_0012">10.1142/9789813279827_0012</a></p>
</div>
<div id="ref-1Av3x1pnk">
<p>31. <strong>A novel multi-network approach reveals tissue-specific cellular modulators of fibrosis in systemic sclerosis</strong><br />
Jaclyn N. Taroni, Casey S. Greene, Viktor Martyanov, Tammara A. Wood, Romy B. Christmann, Harrison W. Farber, Robert A. Lafyatis, Christopher P. Denton, Monique E. Hinchcliff, Patricia A. Pioli, … Michael L. Whitfield<br />
<em>Genome Medicine</em> (2017-03-23) <a href="https://doi.org/gfsptx">https://doi.org/gfsptx</a><br />
DOI: <a href="https://doi.org/10.1186/s13073-017-0417-1">10.1186/s13073-017-0417-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28330499">28330499</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5363043">PMC5363043</a></p>
</div>
<div id="ref-r7HvVKdn">
<p>32. <strong>Meta-analysis of airway epithelium gene expression in asthma.</strong><br />
Yi-Hsuan Tsai, Joel S Parker, Ivana V Yang, Samir NP Kelada<br />
<em>The European respiratory journal</em> (2018-05-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29650561">https://www.ncbi.nlm.nih.gov/pubmed/29650561</a><br />
DOI: <a href="https://doi.org/10.1183/13993003.01962-2017">10.1183/13993003.01962-2017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29650561">29650561</a></p>
</div>
<div id="ref-yKr4neGw">
<p>33. <strong>Identification of a robust methylation classifier for cutaneous melanoma diagnosis</strong><br />
Kathleen Conway, Sharon N. Edmiston, Joel S. Parker, Pei Fen Kuan, Yi-Hsuan Tsai, Pamela A. Groben, Daniel C. Zedek, Glynis A. Scott, Eloise A. Parrish, Honglin Hao, … Nancy E. Thomas<br />
<em>Journal of Investigative Dermatology</em> (2018-12) <a href="https://doi.org/gfsvbj">https://doi.org/gfsvbj</a><br />
DOI: <a href="https://doi.org/10.1016/j.jid.2018.11.024">10.1016/j.jid.2018.11.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30529013">30529013</a></p>
</div>
<div id="ref-9sfG2U7T">
<p>34. <strong>Transcriptome-wide identification and study of cancer-specific splicing events across multiple tumors.</strong><br />
Yihsuan S Tsai, Daniel Dominguez, Shawn M Gomez, Zefeng Wang<br />
<em>Oncotarget</em> (2015-03-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25749525">https://www.ncbi.nlm.nih.gov/pubmed/25749525</a><br />
DOI: <a href="https://doi.org/10.18632/oncotarget.3145">10.18632/oncotarget.3145</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25749525">25749525</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466652">PMC4466652</a></p>
</div>
<div id="ref-jk5tACP8">
<p>35. <strong>Prevalent RNA recognition motif duplication in the human genome.</strong><br />
Yihsuan S Tsai, Shawn M Gomez, Zefeng Wang<br />
<em>RNA (New York, N.Y.)</em> (2014-03-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24667216">https://www.ncbi.nlm.nih.gov/pubmed/24667216</a><br />
DOI: <a href="https://doi.org/10.1261/rna.044081.113">10.1261/rna.044081.113</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24667216">24667216</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3988571">PMC3988571</a></p>
</div>
<div id="ref-6zDUsoSA">
<p>36. <strong>Integrative Genomics-Based Discovery of Novel Regulators of the Innate Antiviral Response.</strong><br />
Robin van der Lee, Qian Feng, Martijn A Langereis, Rob Ter Horst, Radek Szklarczyk, Mihai G Netea, Arno C Andeweg, Frank JM van Kuppeveld, Martijn A Huynen<br />
<em>PLoS computational biology</em> (2015-10-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26485378">https://www.ncbi.nlm.nih.gov/pubmed/26485378</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1004553">10.1371/journal.pcbi.1004553</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26485378">26485378</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4618338">PMC4618338</a></p>
</div>
<div id="ref-Myj0Ouv5">
<p>37. <strong>Genome-scale detection of positive selection in nine primates predicts human-virus evolutionary conflicts.</strong><br />
Robin van der Lee, Laurens Wiel, Teunis JP van Dam, Martijn A Huynen<br />
<em>Nucleic acids research</em> (2017-10-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28977405">https://www.ncbi.nlm.nih.gov/pubmed/28977405</a><br />
DOI: <a href="https://doi.org/10.1093/nar/gkx704">10.1093/nar/gkx704</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28977405">28977405</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5737536">PMC5737536</a></p>
</div>
<div id="ref-3NKcTIrb">
<p>38. <strong>Pharmacogenomics knowledge for personalized medicine.</strong><br />
M Whirl-Carrillo, EM McDonagh, JM Hebert, L Gong, K Sangkuhl, CF Thorn, RB Altman, TE Klein<br />
<em>Clinical pharmacology and therapeutics</em> (2012-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22992668">https://www.ncbi.nlm.nih.gov/pubmed/22992668</a><br />
DOI: <a href="https://doi.org/10.1038/clpt.2012.96">10.1038/clpt.2012.96</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22992668">22992668</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3660037">PMC3660037</a></p>
</div>
<div id="ref-1EYG4hAqR">
<p>39. <strong>CPIC: Clinical Pharmacogenetics Implementation Consortium of the Pharmacogenomics Research Network.</strong><br />
MV Relling, TE Klein<br />
<em>Clinical pharmacology and therapeutics</em> (2011-01-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21270786">https://www.ncbi.nlm.nih.gov/pubmed/21270786</a><br />
DOI: <a href="https://doi.org/10.1038/clpt.2010.279">10.1038/clpt.2010.279</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21270786">21270786</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3098762">PMC3098762</a></p>
</div>
</div>
<script>
// AnchorJS minified version below.
// Source https://github.com/bryanbraun/anchorjs/blob/064abdd0987f305933ec4982af6d0c1cf2fd0814/anchor.js

/**
 * AnchorJS - v4.0.0 - 2017-06-02
 * https://github.com/bryanbraun/anchorjs
 * Copyright (c) 2017 Bryan Braun; Licensed MIT
 */
!function(A,e){"use strict";"function"==typeof define&&define.amd?define([],e):"object"==typeof module&&module.exports?module.exports=e():(A.AnchorJS=e(),A.anchors=new A.AnchorJS)}(this,function(){"use strict";function A(A){function e(A){A.icon=A.hasOwnProperty("icon")?A.icon:"",A.visible=A.hasOwnProperty("visible")?A.visible:"hover",A.placement=A.hasOwnProperty("placement")?A.placement:"right",A.class=A.hasOwnProperty("class")?A.class:"",A.truncate=A.hasOwnProperty("truncate")?Math.floor(A.truncate):64}function t(A){var e;if("string"==typeof A||A instanceof String)e=[].slice.call(document.querySelectorAll(A));else{if(!(Array.isArray(A)||A instanceof NodeList))throw new Error("The selector provided to AnchorJS was invalid.");e=[].slice.call(A)}return e}function n(){if(null===document.head.querySelector("style.anchorjs")){var A,e=document.createElement("style");e.className="anchorjs",e.appendChild(document.createTextNode("")),void 0===(A=document.head.querySelector('[rel="stylesheet"], style'))?document.head.appendChild(e):document.head.insertBefore(e,A),e.sheet.insertRule(" .anchorjs-link {   opacity: 0;   text-decoration: none;   -webkit-font-smoothing: antialiased;   -moz-osx-font-smoothing: grayscale; }",e.sheet.cssRules.length),e.sheet.insertRule(" *:hover > .anchorjs-link, .anchorjs-link:focus  {   opacity: 1; }",e.sheet.cssRules.length),e.sheet.insertRule(" [data-anchorjs-icon]::after {   content: attr(data-anchorjs-icon); }",e.sheet.cssRules.length),e.sheet.insertRule(' @font-face {   font-family: "anchorjs-icons";   src: url(data:n/a;base64,AAEAAAALAIAAAwAwT1MvMg8yG2cAAAE4AAAAYGNtYXDp3gC3AAABpAAAAExnYXNwAAAAEAAAA9wAAAAIZ2x5ZlQCcfwAAAH4AAABCGhlYWQHFvHyAAAAvAAAADZoaGVhBnACFwAAAPQAAAAkaG10eASAADEAAAGYAAAADGxvY2EACACEAAAB8AAAAAhtYXhwAAYAVwAAARgAAAAgbmFtZQGOH9cAAAMAAAAAunBvc3QAAwAAAAADvAAAACAAAQAAAAEAAHzE2p9fDzz1AAkEAAAAAADRecUWAAAAANQA6R8AAAAAAoACwAAAAAgAAgAAAAAAAAABAAADwP/AAAACgAAA/9MCrQABAAAAAAAAAAAAAAAAAAAAAwABAAAAAwBVAAIAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAMCQAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAg//0DwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAAIAAAACgAAxAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADAAAAAIAAgAAgAAACDpy//9//8AAAAg6cv//f///+EWNwADAAEAAAAAAAAAAAAAAAAACACEAAEAAAAAAAAAAAAAAAAxAAACAAQARAKAAsAAKwBUAAABIiYnJjQ3NzY2MzIWFxYUBwcGIicmNDc3NjQnJiYjIgYHBwYUFxYUBwYGIwciJicmNDc3NjIXFhQHBwYUFxYWMzI2Nzc2NCcmNDc2MhcWFAcHBgYjARQGDAUtLXoWOR8fORYtLTgKGwoKCjgaGg0gEhIgDXoaGgkJBQwHdR85Fi0tOAobCgoKOBoaDSASEiANehoaCQkKGwotLXoWOR8BMwUFLYEuehYXFxYugC44CQkKGwo4GkoaDQ0NDXoaShoKGwoFBe8XFi6ALjgJCQobCjgaShoNDQ0NehpKGgobCgoKLYEuehYXAAAADACWAAEAAAAAAAEACAAAAAEAAAAAAAIAAwAIAAEAAAAAAAMACAAAAAEAAAAAAAQACAAAAAEAAAAAAAUAAQALAAEAAAAAAAYACAAAAAMAAQQJAAEAEAAMAAMAAQQJAAIABgAcAAMAAQQJAAMAEAAMAAMAAQQJAAQAEAAMAAMAAQQJAAUAAgAiAAMAAQQJAAYAEAAMYW5jaG9yanM0MDBAAGEAbgBjAGgAbwByAGoAcwA0ADAAMABAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAP) format("truetype"); }',e.sheet.cssRules.length)}}this.options=A||{},this.elements=[],e(this.options),this.isTouchDevice=function(){return!!("ontouchstart"in window||window.DocumentTouch&&document instanceof DocumentTouch)},this.add=function(A){var i,o,s,c,r,a,h,l,u,d,f,g,p=[];if(e(this.options),"touch"===(g=this.options.visible)&&(g=this.isTouchDevice()?"always":"hover"),A||(A="h2, h3, h4, h5, h6"),0===(i=t(A)).length)return this;for(n(),o=document.querySelectorAll("[id]"),s=[].map.call(o,function(A){return A.id}),r=0;r<i.length;r++)if(this.hasAnchorJSLink(i[r]))p.push(r);else{if(i[r].hasAttribute("id"))c=i[r].getAttribute("id");else if(i[r].hasAttribute("data-anchor-id"))c=i[r].getAttribute("data-anchor-id");else{u=l=this.urlify(i[r].textContent),h=0;do{void 0!==a&&(u=l+"-"+h),a=s.indexOf(u),h+=1}while(-1!==a);a=void 0,s.push(u),i[r].setAttribute("id",u),c=u}d=c.replace(/-/g," "),(f=document.createElement("a")).className="anchorjs-link "+this.options.class,f.href="#"+c,f.setAttribute("aria-label","Anchor link for: "+d),f.setAttribute("data-anchorjs-icon",this.options.icon),"always"===g&&(f.style.opacity="1"),""===this.options.icon&&(f.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(f.style.lineHeight="inherit")),"left"===this.options.placement?(f.style.position="absolute",f.style.marginLeft="-1em",f.style.paddingRight="0.5em",i[r].insertBefore(f,i[r].firstChild)):(f.style.paddingLeft="0.375em",i[r].appendChild(f))}for(r=0;r<p.length;r++)i.splice(p[r]-r,1);return this.elements=this.elements.concat(i),this},this.remove=function(A){for(var e,n,i=t(A),o=0;o<i.length;o++)(n=i[o].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[o]))&&this.elements.splice(e,1),i[o].removeChild(n));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var t=/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\]/g;return this.options.truncate||e(this.options),A.trim().replace(/\'/gi,"").replace(t,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&(" "+A.firstChild.className+" ").indexOf(" anchorjs-link ")>-1,t=A.lastChild&&(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ")>-1;return e||t||!1}}return A});

// Enable links for selected headers
var anchors = new AnchorJS();
anchors.add("h2, h3, h4")
</script>
<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
<script src="https://hypothes.is/embed.js" async></script>
</body>
</html>
